docs/conf.py

```diff
@@ -110,7 +110,6 @@
 # directories to ignore when looking for source files.
 exclude_patterns = [
     "_build",
-    "**/.nox/**/*",
```

Comment:
I suspect owlbot will be adding this back soon. But I'm fine not fighting the bot on this one.